src/platforms/dart/config.yml

```
@@ -0,0 +1,4 @@
+title: Dart
+caseStyle: canonical
```

Options are showing as `before-breadcrumb` instead of camel case: `beforeBreadcrumb`

I guess this is the right format:
```suggestion
caseStyle: camelCase
```

---

yeah but Java is `canonical` and I copy-paste the config, I wondered this too

---

fixed java/android too